codec_test.go

```diff
@@ -747,6 +747,47 @@ var disassembleResponseStruct = DisassembleResponse{
 	},
 }
 
+// -------- Custom Request/Response --------
```

Comment:
nit: Can we move this section below the events section as well?

Reply:
Done.